test/types/enum/ferguson/dependent-uint.compopts

```diff
@@ -0,0 +1 @@
+--ignore-errors-for-pass
```

Comment:
I don't think I'm familiar with `--ignore-errors-for-pass`. For this test though I think it'd be better if we just remove the is_uint calls since which type they coerce to is now moot. The enum itself is still worthy of testing since it contains large numbers in an unusual manner.